app/src/main/java/com/wednesday/banklocator/adapter/IfscAdapter.kt

```diff
@@ -31,9 +31,14 @@ class IfscAdapter(var item: IfscResponse) : RecyclerView.Adapter<IfscAdapter.Art
     }
 
     override fun getItemCount(): Int {
-        if (item != null) {
+        if (item.BANK != "Search") {
```

Comment:
Why is this needed?

Reply:
I was facing some problems in returning the size of the items i.e. why I've initialized "item.BANK" to "search".